### DISCUSSION TITLE
Open file in new "view" when in splitted view?

### DISCUSSION THREAD

**Discussion by mkastner** [helix-editor/helix#5257] [Q&A] [answered]
Let's say I have a vertically splitted "view" with two different buffers and the cursor is in the right buffer.

Now I'd like to open a new buffer without replacing the right side of the splitted view.

How can create a new view with only the new content without losing the current splitted view with the current buffers?

**Comment by the-mikedavis** [comment, marked as answer]
This would be covered by tabs which aren't implemented yet: https://github.com/helix-editor/helix/issues/2295

**Comment by mkastner** [comment]
Ooops, sure, that would be tabs, that's right. It would really be nice to have tabs, but it's not that big a deal. Helix is already awesome!
